earthmover/include/starter_project/sources/attendance.tsv

```diff
@@ -0,0 +1,200 @@
+day	section	student_id	absent
```

Comment:
This example data (and corresponding template) is Ed-Fi specific. I'd prefer to keep earthmover more general, and disconnected from lightbeam and Ed-Fi, to make it more accessible to users coming from other domains. (We can debate this.)

In the meantime:
1. this example data file seems too large to include in every earthmover installation? perhaps ~10 rows would be plenty
2. by default, [Python's setuptools module excludes data files from packages](https://setuptools.pypa.io/en/latest/userguide/datafiles.html), however [here](https://github.com/edanalytics/earthmover/blob/9bb733a7c7f1e5cc08e1e6995603e317e1c4e4ff/MANIFEST.in#L1) we tell it to include in earthmover the data files used by the test suite... would it make sense to copy one of the data files from the test suite (and possibly a JSON template too) for this `init` project instead of including (another) different file?

Reply:
Ah yeah, I failed to notice the test files. That all seems fair to me, I'll make the change